manifest.json

```diff
@@ -22,18 +22,33 @@
       "default": "SUM",
       "required": false
     },
+    "filterExpression": {
+      "type": "string",
+      "description": "An AQL expression used to filter data. The current document is called 'doc'.",
+      "required": false
+    },
     "dateField": {
       "type": "string",
-      "description": "Name of the field containing the date time for each data point.",
+      "description": "Name of the field containing the date time for each data point. Either set 'dateField' or 'dateExpression'.",
```

Comment:
I'd elaborate that the `*Expression` configs take precedence if both are specified.

Reply:
done